### PR TITLE
Updated email_goody opt in text

### PR DESCRIPTION
**Pull request by MyIgel**
The current text is somewhat hard to parse, that version is a bit better and underlines the possibility that there might not be any vouchers at all.
Wording alternatives welcome ;)

### CODE REVIEW

**Comment by xuwhite** [comment]
"bis dahin" ist eine sehr ungenaue aussage
pls reword to be more specific

**Comment by MyIgel** [comment]
@xuwhite Do you have any more specific wording ideas for that? ;)

File: resources/lang/de_DE/default.po [hunk 1]
```diff
@@ -2163,8 +2157,8 @@ msgid "settings.profile.email_by_human_allowed"
 msgstr "Erlaube Himmel-Engeln dich per Mail zu kontaktieren."
 
 msgid "settings.profile.email_goody"
-msgstr "Um Voucher zu erhalten, stimme zu, dass Nick, E-Mail-Adresse, geleistete Arbeit und Shirtgröße "
-"bis zum nächsten gleichartigen Event gespeichert werden."
+msgstr "Um gegebenenfalls Voucher für das nächste gleichartige Event zu erhalten, stimme Ich zu, "
```
Comment (weeman1337) [inline resources/lang/de_DE/default.po:2160]:
```suggestion
msgstr "Um gegebenenfalls Voucher für das nächste gleichartige Event zu erhalten stimme ich zu, "
```

File: resources/lang/de_DE/default.po [hunk 2]
```diff
@@ -2163,8 +2157,8 @@ msgid "settings.profile.email_by_human_allowed"
...
+"dass mein Nick, E-Mail-Adresse, geleistete Arbeit und Shirtgröße bis da hin gespeichert werden."
```
Comment (weeman1337) [inline resources/lang/de_DE/default.po:2161]:
```suggestion
"dass mein Nick, E-Mail-Adresse, geleistete Arbeit und Shirt-Größe bis da hin gespeichert werden."
```

File: resources/lang/en_US/default.po [hunk 3]
```diff
@@ -281,8 +281,8 @@ msgid "settings.profile.email_by_human_allowed"
 msgstr "Allow heaven angels to contact you by e-mail."
 
 msgid "settings.profile.email_goody"
-msgstr "To receive vouchers, give consent that nick, e-mail address, worked hours and shirt size will be stored until "
-"the next similar event."
+msgstr "To potentially receive vouchers for the next similar event, I consent "
```
Comment (weeman1337) [inline resources/lang/en_US/default.po:284]:
```suggestion
msgstr "To possibly receive vouchers for the next similar event, I consent "
```

**Review by weeman1337 — COMMENTED** [review]
Looks good, some smaller suggestions.
